tests/gtest/avif_fuzztest_helpers.h

```diff
@@ -167,13 +168,31 @@ inline auto ArbitraryAvifAnim() {
   return fuzztest::OneOf(ArbitraryAvifAnim8b(), ArbitraryAvifAnim16b());
 }
 
-// TODO: Try StructOf<Metadata>(StructOf<uint32_t[3]>())?
+// Generates two signed fractions where the first one is smaller than the second
+// one.
+inline auto ArbitraryMinMaxSignedFraction() {
+  return fuzztest::FlatMap(
+      [](int32_t max_n, uint32_t max_d) {
+        return fuzztest::Map(
+            [max_n, max_d](int32_t min_n) {
+              // For simplicity, use the same denominator for both fractions.
+              // This does not cover all possible fractions but makes it easy
+              // to gurantee that the fraction is smaller.
+              return std::pair<avifSignedFraction, avifSignedFraction>(
+                  {min_n, max_d}, {max_n, max_d});
+            },
+            fuzztest::InRange<int32_t>(std::numeric_limits<int32_t>::min(),
+                                       max_n));
+      },
+      fuzztest::Arbitrary<int32_t>(),
+      fuzztest::InRange<uint32_t>(1, std::numeric_limits<uint32_t>::max()));
```

Comment:
Optional: Use `fuzztest::NonZero<uint32_t>()` instead?

If this change is correct, make the same change to the other instances below. Or you can make these change in a separate pull request.